library/billing.inc

```diff
@@ -229,40 +253,56 @@ function updateClaim(
         "x12_partner_id = '$partner_id'";
         ****/
         sqlBeginTrans();
-        $version = sqlQuery("SELECT IFNULL(MAX(version),0) + 1 AS increment FROM claims WHERE patient_id = ? AND encounter_id = ?", array( $patient_id, $encounter_id ));
+        $sqlBindArray = array();
+        $sqlBindArray[] = $patient_id;
+        $sqlBindArray[] = $encounter_id;
+        $version = sqlQuery("SELECT IFNULL(MAX(version),0) + 1 AS increment FROM claims WHERE patient_id = ? AND encounter_id = ?", $sqlBindArray);
+
+        $sqlBindArray = array();
+        $sqlBindArray[] = $patient_id;
+        $sqlBindArray[] = $encounter_id;
         if ($crossover<>1) {
             $sql .= "INSERT INTO claims SET " .
-              "patient_id = $patient_id, " .
-              "encounter_id = $encounter_id, " .
+              "patient_id = ?, " .
+              "encounter_id = ?, " .
               "bill_time = NOW() $claimset ," .
               "version = " . $version['increment'];
+            $sqlBindArray = array_merge($sqlBindArray, $sqlBindClaimset);
         } else {//Claim automatic forward case.startTra
             $sql= "INSERT INTO claims SET " .
-              "patient_id = $patient_id, " .
-              "encounter_id = $encounter_id, " .
-              "bill_time = NOW(), status=$status ," .
+              "patient_id = ?, " .
+              "encounter_id = ?, " .
+              "bill_time = NOW(), status=? ," .
               "version = " . $version['increment'];
+            $sqlBindArray[] = $status;
```

Comment:
Change above to:
``````
array_push($sqlBindArray, $patient_id, $encounter_id, $status, $version['increment']);
``````
and bind the $version['increment'] variable in above sql call